### PR TITLE
[zh-cn]sync managing-secret-using-config-file.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh-cn]sync managing-secret-using-config-file.md

### CODE REVIEW

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 1]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+-->
+1. 将清单应用到你的集群：
+
+   ```shell
+   kubectl apply -f ./secret.yaml
+   ```
+
+<!--
+   The output is similar to:
+-->
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:320]:
```suggestion
   <!--
   The output is similar to:
   -->
```

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 2]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+1. Encode the new password string:
+-->
+## 编辑 Secret {#edit-secret}
+
+要编辑使用清单创建的 Secret 中的数据，请修改清单中的 `data` 或 `stringData` 字段并将文件应用到集群。
+你可以编辑现有的 `Secret` 对象，除非它是[不可变的](/zh/docs/concepts/configuration/secret/#secret-immutable)。
+
+例如，如果你想将上一个示例中的密码更改为 `birdsarentreal`，请执行以下操作：
+
+1. 编码新密码字符串
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:278]:
```suggestion
1. 编码新密码字符串：
```

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 3]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+[immutable](/docs/concepts/configuration/secret/#secret-immutable).
+
+For example, if you want to change the password from the previous example to
+`birdsarentreal`, do the following:
+
+1. Encode the new password string:
+-->
+## 编辑 Secret {#edit-secret}
+
+要编辑使用清单创建的 Secret 中的数据，请修改清单中的 `data` 或 `stringData` 字段并将文件应用到集群。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:273]:
```suggestion
要编辑使用清单创建的 Secret 中的数据，请修改清单中的 `data` 或 `stringData` 字段并将此清单文件应用到集群。
```

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 4]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+   输出类似于：
+
+   ```
+   YmlyZHNhcmVudHJlYWw=
+   ```
+
+<!--
+1. Update the `data` field with your new password string:
+-->
+1. 使用你的新密码字符串更新 `data` 字段：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:296]:
```suggestion
2. 使用你的新密码字符串更新 `data` 字段：
```

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 5]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+   type: Opaque
+   data:
+     username: YWRtaW4=
+     password: YmlyZHNhcmVudHJlYWw=
+   ```
+
+<!--
+1. Apply the manifest to your cluster:
+-->
+1. 将清单应用到你的集群：
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:312]:
```suggestion
3. 将清单应用到你的集群：
```

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 6]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+Kubernetes updates the existing `Secret` object. In detail, the `kubectl` tool
+notices that there is an existing `Secret` object with the same name. `kubectl`
+fetches the existing object, plans changes to it, and submits the changed
+`Secret` object to your cluster control plane.
+
+If you specified `kubectl apply --server-side` instead, `kubectl` uses
+[Server Side Apply](/docs/reference/using-api/server-side-apply/) instead.
+-->
+Kubernetes 更新现有的 `Secret` 对象。详细地说，`kubectl` 工具发现存在一个同名的现有 `Secret` 对象。
+`kubectl` 获取现有对象，计划对其进行更改，并将更改后的 `Secret` 对象提交到您的集群控制平面。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:337]:
```suggestion
`kubectl` 获取现有对象，计划对其进行更改，并将更改后的 `Secret` 对象提交到你的集群控制平面。
```

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 7]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+
+如果你指定了 `kubectl apply --server-side`，则 `kubectl` 使用
+[Server Side Apply](/zh-cn/docs/reference/using-api/server-side-apply/) 代替。
```
Comment (windsonsea) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:340]:
```suggestion
[服务器端应用（Server-Side Apply）](/zh-cn/docs/reference/using-api/server-side-apply/) 代替。
```
It's better to keep consistent with the title after you clicked.

**Review by windsonsea — COMMENTED** [review]
Find some alignment and bullets errors.

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 8]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+
+1. 编码新密码字符串：
+
+   ```shell
+   echo -n 'birdsarentreal' | base64
+   ```
+
+<!--
+   The output is similar to:
+-->
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:286]:
```suggestion
   <!--
   The output is similar to:
   -->
```

**Comment by Sea-n** [comment]
Hey @my-git9,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by my-git9** [comment]
> Hey @my-git9,
> 
> It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

Sorry, I forgot this PR before and have updated it as suggested

File: content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 9]
```diff
@@ -255,6 +255,90 @@ type: Opaque
...
+<!--
+Kubernetes updates the existing `Secret` object. In detail, the `kubectl` tool
+notices that there is an existing `Secret` object with the same name. `kubectl`
+fetches the existing object, plans changes to it, and submits the changed
+`Secret` object to your cluster control plane.
+
+If you specified `kubectl apply --server-side` instead, `kubectl` uses
+[Server Side Apply](/docs/reference/using-api/server-side-apply/) instead.
+-->
+Kubernetes 更新现有的 `Secret` 对象。详细地说，`kubectl` 工具发现存在一个同名的现有 `Secret` 对象。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/configmap-secret/managing-secret-using-config-file.md:336]:
```suggestion
Kubernetes 更新现有的 `Secret` 对象。具体而言，`kubectl` 工具发现存在一个同名的现有 `Secret` 对象。
```

**Review by tengqm — COMMENTED** [review]
LGTM other than one suggestion.
/approve

**Pull request merged**